File: clients/dotnet/models/IsoCountryModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// Represents an ISO 3166 recognized country
    /// </summary>
    public class IsoCountryModel
    {
        /// <summary>
        /// The two character ISO 3166 country code
        /// </summary>
        public String code { get; set; }

        /// <summary>
        /// The full name of this country as it is known in US English
        /// </summary>
        public String name { get; set; }

        /// <summary>
        /// True if this country is a member of the European Union
        /// </summary>
        public Boolean? isEuropeanUnion { get; set; }


    }
}

File: clients/php/models/IsoCountryModel.php
<?php 
namespace Avalara;
/*
 * AvaTax API Client Library
 *
 * (c) 2004-2016 Avalara, Inc.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 *
 * @category   AvaTax client libraries
 * @package    Avalara.AvaTaxClient
 * @author     Ted Spence <ted.spence@avalara.com>
 * @author     Bob Maidens <bob.maidens@avalara.com>
 * @copyright  2004-2016 Avalara, Inc.
 * @license    https://www.apache.org/licenses/LICENSE-2.0
 * @version    2.16.12-30
 * @link       https://github.com/avadev/AvaTaxClientLibrary
 */


/**
 * Represents an ISO 3166 recognized country
 */
final class IsoCountryModel
{
    /**
     * @var String The two character ISO 3166 country code
     */
    public $code;

    /**
     * @var String The full name of this country as it is known in US English
     */
    public $name;

    /**
     * @var Boolean? True if this country is a member of the European Union
     */
    public $isEuropeanUnion;

}
